examples/mnist-inference-web/build.rs

```diff
@@ -0,0 +1,26 @@
+use std::fs;
```

Comment:
I guess this file will not be necessary when the model state will be saved using the bincode format!